gui/wxpython/startup/utils.py

```diff
@@ -55,6 +58,47 @@ def get_possible_database_path():
     return path
 
 
+def create_database_directory():
+    """Creates the standard GRASS GIS directory.
+
+    Creates database directory named grassdata in the standard location 
+    according to the platform.  
+    
+    Returns the new path as a string or None if nothing was found or created.
+    """
+    home = os.path.expanduser('~')     
+   
+    # Candidate for case independent "grassdata" for Linux and macOS
+    if sys.platform.startswith('linux') or sys.platform.startswith('darwin'):
+        candidate = os.path.join(home, "grassdata")
+    # Candidates for case independent "grassdata" in other dirs (Windows)
+    else:  
+        candidate = os.path.join(home, "Documents", "grassdata")
+        
+    # Create "grassdata" directory                          
+    try:
+        os.mkdir(candidate)
+        return candidate
+    except OSError:
+        pass
+
+    # Temporary "grassdata" directory     
+    tmp = os.path.join(
+        tempfile.gettempdir(),
+        "grassdata_{}".format(getpass.getuser())
+    ) 
+    
+    # Check if exists, if does not, create it
```

Comment:
The intended behavior may not be clear from the code, so add another comment saying something like "The created tmp is not cleaned by GRASS, so we are relying on the system to do it at some point. The positive outcome is that another GRASS instance will find the data created by the first one which is desired in the "try out GRASS" use case we are aiming towards." See the original ticket for alternative formulation of this.